src/editor/sidebar/index.js

```diff
@@ -13,100 +13,203 @@ import { PluginDocumentSettingPanel } from '@wordpress/edit-post';
 import './style.scss';
 
 const SidebarComponent = props => {
-	if ( props.postType !== 'newspack_spnsrs_cpt' ) {
-		return null;
-	}
-
-	const { meta, title, updateMetaValue } = props;
+	const { meta, postType, title, updateMetaValue } = props;
 	const {
 		newspack_sponsor_url,
 		newspack_sponsor_flag_override,
 		newspack_sponsor_byline_prefix,
 		newspack_sponsor_sponsorship_scope,
+		newspack_sponsor_native_byline_display,
+		newspack_sponsor_native_category_display,
+		newspack_sponsor_underwriter_style,
+		newspack_sponsor_underwriter_placement,
 		newspack_sponsor_only_direct,
 		newspack_sponsor_disclaimer_override,
 	} = meta;
 	const { settings, defaults } = window.newspack_sponsors_data;
+	const isSponsor = 'newspack_spnsrs_cpt' === postType; // True if the post being edited is a sponsor, false if a post type that can be sponsored.
+
+	// If the post is not a sponsor but a post that can be sponsored, add default options to inherit values from the sponsor.
+	const scopeDefault = isSponsor ? 'native' : '';
+	const scopeOptions = [
+		{ value: 'native', label: __( 'Native content', 'newspack-sponsors' ) },
+		{ value: 'underwritten', label: __( 'Underwritten content', 'newspack-sponsors' ) },
+	];
+	const bylineDefault = isSponsor ? 'sponsor' : 'inherit';
+	const bylineOptions = [
+		{ value: 'sponsor', label: __( 'Sponsor only', 'newspack-sponsors' ) },
+		{ value: 'author', label: __( 'Both sponsor and author byline', 'newspack-sponsors' ) },
+	];
+	const categoryDefault = isSponsor ? 'sponsor' : 'inherit';
+	const categoryOptions = [
+		{ value: 'sponsor', label: __( 'Sponsor only', 'newspack-sponsors' ) },
+		{ value: 'category', label: __( 'Sponsor and categories', 'newspack-sponsors' ) },
+	];
+	const underwriterStyleDefault = isSponsor ? 'standard' : 'inherit';
+	const underwriterStyleOptions = [
+		{ value: 'standard', label: __( 'Standard', 'newspack-sponsors' ) },
+		{ value: 'simple', label: __( 'Simple', 'newspack-sponsors' ) },
+	];
+	const underwriterPlacementDefault = isSponsor ? 'top' : 'inherit';
+	const underwriterPlacementOptions = [
+		{ value: 'top', label: __( 'Top', 'newspack-sponsors' ) },
+		{ value: 'bottom', label: __( 'Bottom', 'newspack-sponsors' ) },
+	];
+	if ( ! isSponsor ) {
+		const defaultOption = {
+			value: 'inherit',
+			label: __( 'Inherit from sponsor', 'newspack-sponsors' ),
+		};
+		scopeOptions.unshift( defaultOption );
+		bylineOptions.unshift( defaultOption );
+		categoryOptions.unshift( defaultOption );
+		underwriterStyleOptions.unshift( defaultOption );
+		underwriterPlacementOptions.unshift( defaultOption );
+	}
 
 	return (
 		<PluginDocumentSettingPanel
 			className="newspack-sponsors"
 			name="newspack-sponsors"
 			title={ __( 'Sponsor Settings', 'newspack-sponsors' ) }
```

Comment:
This took a little refactoring, but 638e61c moves the "Sponsor Settings" options to the "Sponsors" panel that's used for assigning sponsors to supported post types. It's still rendered as its own sidebar panel on sponsor post types, since the same component is used to render the settings for the sponsor itself. Additionally, the settings UI is only shown if at least one sponsor is assigned to the post.